interop_binaries/setup.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+until pg_isready -U postgres; do sleep 1; done
```

Comment:
This will wait for postgres forever, right? I think we should give up after some delay so that e.g. postgres crashing on startup due to misconfiguration doesn't cause a test to hang forever.